plaso/output/elastic_ts.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+"""An output module that saves events to Elasticsearch TS index."""
+
+from plaso.output import logger
+from plaso.output import manager
+from plaso.output import shared_elastic
+
+
+class ElasticTimesketchOutputModule(
```

Comment:
Please add a test for this output module also see elastic test https://github.com/log2timeline/plaso/blob/master/tests/output/elastic.py

Reply:
ack, done